src/test/java/seedu/address/model/event/UniqueEventListTest.java

```diff
@@ -58,6 +60,12 @@ public void add_duplicateEvent_throwsDuplicateEventException() {
         assertThrows(DuplicateEventException.class, () -> uniqueEventList.add(BIRTHDAY_PARTY));
     }
 
+    @Test
+    public void addEvent_invalidDateTimeRange_throwsInvalidDateTimeRangeException() {
+        assertThrows(InvalidDateTimeRangeException.class, () -> uniqueEventList.add(
+                        new EventBuilder(BIRTHDAY_PARTY).withEndDateAndTime(VALID_START_DATE_TIME_TUTORIAL).build()));
+    }
+
```

Comment:
Can you add one more test case for editing an event but with invalid date time range?